monkey/infection_monkey/exploit/tests/zerologon_utils/test_vuln_assessment.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from nmb.NetBIOS import NetBIOS
+
+from infection_monkey.exploit.zerologon_utils.vuln_assessment import \
+    get_dc_details
+from infection_monkey.model.host import VictimHost
+
+
+DOMAIN_NAME = "domain-name"
+IP = "0.0.0.0"
+
+
+@pytest.fixture
+def host():
+    return VictimHost(IP, DOMAIN_NAME)
+
+
+def test_get_dc_details_multiple_netbios_names(host, monkeypatch):
+    def mock_queryIPForName(*args, **kwargs):
```

Comment:
This is technically a stub, not a mock: https://martinfowler.com/bliki/TestDouble.html